docs/accessing_mgmt_tools.md

```diff
@@ -0,0 +1,54 @@
+## Accessing the Management Tools in MARA
+
+Currently, the management tool suite in MARA consists of:
+
+- Prometheus
+- Grafana
+- Locust
+- Elasticsearch
+- Kibana
```

Comment:
Would be cool to make these links out to the tool homepages in case some of them are not familiar.

Reply:
Oversight on my part, done.